src/routes/chat/ChatHeader/ChatHeader.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import ChatCreateOffer from './ChatCreateOfferHeader';
+import ChatDefaultHeader from './ChatDefaultHeader';
+import ChatOfferDetailsHeader from './ChatOfferDetailsHeader';
+import './ChatHeader.scss';
+
+const ChatHeader = () => {
+  const [header, setHeader] = useState(0);
```

Comment:
CONFUSING NAMES. I suggest `const [isMakeOfferHeaderShown, setIsMakeOfferHeaderShown] = useState(false);` instead.

Reply:
Why did you use integers when there are booleans anyways lol...